pkg/projects/Microsoft.NETCore.UniversalWindowsPlatform.CoreCLR/files/Microsoft.NETCore.UniversalWindowsPlatform.CoreCLR.targets

```diff
@@ -0,0 +1,14 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <BuildDependsOn>
+      CheckCoreRuntimeCompatibility;
+      $(BuildDependsOn);
+    </BuildDependsOn>
+  </PropertyGroup>
+
+  <Target Name="CheckCoreRuntimeCompatibility">
+    <Error
```

Comment:
Can I get your thoughts on this piece? This targets file will kick in when building the project in VS and will currently throw an error when targeting an existing Win 10 SDK (i.e. this can only be used with future Win 10 SDK's). Are the error condition and the message okay?